mmv1/templates/terraform/examples/bigquery_connection_kms.tf.erb

```diff
@@ -0,0 +1,24 @@
+resource "google_bigquery_connection" "<%= ctx[:primary_resource_id] %>" {
+   connection_id = "<%= ctx[:vars]['connection_id'] %>"
+   location      = "aws-us-east-1"
+   friendly_name = "👋"
+   description   = "a riveting description"
+   kms_key_name  = "<%= ctx[:vars]['kms_key_name'] %>"
+   aws { 
```

Comment:
This will likely prevent us from being able to automatically test this feature; could it use cloud spanner instead? Or better yet, could this new field be added to the existing Cloud Spanner example instead? (BigQuery Connection already has a bunch of examples so it would be great to not add another one.)

Reply:
will try